### PR TITLE
Issue #67 Add common installer errors article

### PR DESCRIPTION
**Pull request by vyanovsky**
Add common-installer-errors.md

### CODE REVIEW

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- Uninstall some programs or delete unnecessary files from the drive you were trying to install AdGuard VPN to.
+
+- Download, install, and run [AdwCleaner](http://www.bleepingcomputer.com/download/adwcleaner/), a free piece of software by Malwarebytes. It will, among other things, clean your system from all kinds of extra "leftover" files that remain after incorrectly uninstall programs and such. It will help clean up some disk space.
+
+- Reboot your computer. Sometimes temporary files can take up a considerable amount of disk space, and restarting your PC is the most reliable way to get rid of them.
+
+### Error 1601: Windows Installer is not accessible {#error-1601}
+
+You could say this is a particular sub-case of Error 1603. The possible solutions are similar:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:31]: subtype is better to my mind

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+- Start and re-register Microsoft Installer service. It requires some work.
+
+    1) Press *Win + R* and enter **services.msc**.
+    2) Find in the list and double click on *Windows Installer*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:36]: double-click *Windows Installer*

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    3) Hit *Start* button under *Service status* and hit *OK*. If the service status is **running**, you should click on *Stop* first and then hit *Start*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:37]: Hit *Start*
click *Stop*

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+- Reboot the PC and start the installation all over again. Sometimes that's enough to fix the problem.
+
+### Error 1602: Canceled by user {#error-1602}
+
+If you got this error code, chances are you have interrupted the installation process manually in one way or another. What you can do is:
+
+- Don't close the installer window. When the installation is complete, it will close automatically.
+
+- If a dialogue window pops up during the installation, hit "Yes" to grant the installer the required permissions. Clicking on "No" will cancel he installation.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:49]: Clicking *No*
the installation

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+- Don't start other processes while the installation process is going on.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:51]: while the installation is in progress

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+### Error 1603: Fatal error during installation {#error-1603}
+
+The error sounds scarier than it actually is. In reality, this is a rather generic error that can have many different causes, and some of them are easily fixed. Try the following solutions:
+
+- Press the *Win* key, search for *Command Prompt* and run it. There, type in `sfc /scannow` and press *Enter*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:57]: , and run it

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+- Choose a different installation folder. It is possible that the current installation folder has some access restrictions. Also make sure you don't select an external drive, a virtual drive, etc.
+
+- Uninstall AdGuard VPN using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+
+- Start and re-register Microsoft Installer service. It requires some work.
+
+    1) Press *Win + R* and enter ***services.msc***.
+    2) Find in the list and double click on *Windows Installer*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:66]: same as above

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+    3) Hit *Start* button under *Service status* and hit *OK*. If the service status is **running**, you should click on *Stop* first and then hit *Start*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:67]: same as above

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+    1) Press *Win + R* and enter ***services.msc***.
+    2) Find in the list and double click on *Windows Installer*.
+    3) Hit *Start* button under *Service status* and hit *OK*. If the service status is **running**, you should click on *Stop* first and then hit *Start*. 
+    4) Press *Win + R*, type and enter ***msiexec /unregister*** and hit *Enter*.
+    5) Press *Win + R* again, type and enter ***msiexec /regserver*** and hit *Enter*
+
+- Acquire full permissions on the drive for installation. It is possible that the error 1603 occurs because you don’t have full permissions on the file location. It's also not as easy as some of the other solutions:
+
+    1) Open *File Explorer*, right click on the drive containing the installation location and select *Properties*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:73]: right-click the drive
, and select

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    2) Go to *Security* tab and click on *Edit*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:74]: click *Edit*

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+    3) Single click on *SYSTEM* and ensure that the *Allow* box of every item in *Permissions for SYSTEM* is checked (if it is checkable). Do the same check for *Administrators*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:75]: Click *SYSTEM*

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+    4) Click on *OK* to go back to *Properties* dialog. Then click on *Advanced*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:76]: Click *OK*, click *Advanced*

Comment (el-termikael) [reply docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:76]: please make similar edits below

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+    5) Click on *Change Permissions*.
+    6) On *Permissions* tab, double click on *Administrators*.
+    7) Select *This folder, subfolders and files* for *Applies to* field and tick all the available *Basic permissions*. After that hit *OK*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:79]: for the *Applies to* field

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+
+- Reboot your PC and start the installer again. When you restart the computer, all ongoing processes will stop, including all copies of the installer.
+
+- Don't make multiple clicks on the installer even if doesn't start right away. Sometimes it may take a few seconds to display the installer UI.
+
+### Error 1638: Another version of this product is already installed {#error-1638}
+
+It's very likely that you've already installed AdGuard VPN before.
+
+- Check if AdGuard VPN is already installed on your computer. You can do it by pressing the *Win* key and typing in ***AdGuard VPN***.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:95]: why ***?

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+- Maybe there's some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:97]: there are

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
+---
+title: Common installer errors
+sidebar_position: 2
+---
+
+This article contains some of the most common errors that you can encounter during the AdGuard VPN for Windows installation, and possible ways to solve them.
+
+### Error 5: Access Denied {#error-5}
+
+This error occurs when there's something wrong with permissions. There may be several different reasons why AdGuard VPN installer doesn't have the permissions it requires to properly finish the installation process. You can try the following steps:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:10]: also, try to abstain from using apostrophes in the Knowledge base (there are six instances of them, if I'm not mistaken)

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+
+- Check if AdGuard VPN is already installed on your computer. You can do it by pressing the *Win* key and typing in ***AdGuard VPN***.
+
+- Maybe there's some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+
+### Other errors {#other}
+
+If you've encountered an error that's not listed above, it is possible that we can solve it by ourselves. But in order to do that, we need log files from you. Please perform the following steps:
+
+- Find and archive **AdGuard VPN installation logs** in a same way as it is described in [this article](https://adguard.com/kb/adguard-for-windows/solving-problems/installation-logs/).
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:103]: in the same way that is described

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+
+- Find and save to disk **Event Viewer** logs. [This article](https://adguard.com/kb/adguard-for-windows/solving-problems/system-logs/) explains how to do that.
+
+- Please email all these files from two previous steps to the support team at **support@adguard.com** and describe the problem in the message body. Our tech support agents will reply to you as soon as possible.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:107]: Please email all these files from the two previous steps at **support@adguard.com** and describe the problem in the message body. Our support team will reply to you as soon as possible.

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md (hunk 14 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:95]: don't know, it was like this in the original text

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md (hunk 16 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:10]: hm, okay, I didn't write this original text tho :)

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md (hunk 11 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:75]: We don't see how it looks in the interface, I suppose there is a reason why it deliberately says "single", therefore I suggest "Single-click _SYSTEM_"

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 19]
```diff
@@ -94,14 +93,14 @@ It's very likely that you've already installed AdGuard VPN before.
...
 
-- Maybe there's some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+- Maybe there are some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
 
 ### Other errors {#other}
 
 If you've encountered an error that's not listed above, it is possible that we can solve it by ourselves. But in order to do that, we need log files from you. Please perform the following steps:
 
-- Find and archive **AdGuard VPN installation logs** in a same way as it is described in [this article](https://adguard.com/kb/adguard-for-windows/solving-problems/installation-logs/).
+- Find and archive **AdGuard VPN installation logs** in the same way that it is described in [this article](https://adguard.com/kb/adguard-for-windows/solving-problems/installation-logs/).
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:102]: that is described

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 20]
```diff
@@ -94,14 +93,14 @@ It's very likely that you've already installed AdGuard VPN before.
 
 - Check if AdGuard VPN is already installed on your computer. You can do it by pressing the *Win* key and typing in ***AdGuard VPN***.
 
-- Maybe there's some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+- Maybe there are some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
 
 ### Other errors {#other}
 
 If you've encountered an error that's not listed above, it is possible that we can solve it by ourselves. But in order to do that, we need log files from you. Please perform the following steps:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:100]: you have, that is

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 21]
```diff
@@ -46,15 +45,15 @@ If you got this error code, chances are you have interrupted the installation pr
 
 - Don't close the installer window. When the installation is complete, it will close automatically.
 
-- If a dialogue window pops up during the installation, hit "Yes" to grant the installer the required permissions. Clicking on "No" will cancel he installation.
+- If a dialogue window pops up during the installation, hit "Yes" to grant the installer the required permissions. Clicking "No" will cancel the installation.
 
-- Don't start other processes while the installation process is going on.
+- Don't start other processes while the installation is in progress.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:50]: Do not

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 22]
```diff
@@ -46,15 +45,15 @@ If you got this error code, chances are you have interrupted the installation pr
 
 - Don't close the installer window. When the installation is complete, it will close automatically.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:46]: Do not

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md [hunk 23]
```diff
@@ -3,11 +3,11 @@ title: Common installer errors
 sidebar_position: 2
 ---
 
-This article contains some of the most common errors that you can encounter during the AdGuard VPN for Windows installation, and possible ways to solve them.
+This article contains some of the most common errors you can encounter during the installation of AdGuard VPN for Windows and possible ways to solve them.
 
 ### Error 5: Access Denied {#error-5}
 
-This error occurs when there's something wrong with permissions. There may be several different reasons why AdGuard VPN installer doesn't have the permissions it requires to properly finish the installation process. You can try the following steps:
+This error occurs when there's something wrong with permissions. There may be several different reasons why AdGuard VPN installer does not have the permissions it requires to properly finish the installation process. You can try the following steps:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:10]: there is

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md (hunk 20 above)
Comment (el-termikael) [reply docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:100]: please check the instances with `'`

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md (hunk 20 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md:100]: Changed to "you have", but kept "that's" as it seems more natural and easy to read that way

**Review by el-termikael — APPROVED** [review]

**Pull request merged**